prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -193,6 +205,20 @@ presubmits: # runs on PRs
       path_alias: github.com/kyma-project/test-infra
       base_ref: release-0.8
 
+  - name: pre-rel09-kyma-gke-integration
```

Comment:
when are we going to remove jobs for older releases?

Reply:
Good question. I confirmed with @jose-cortina that we need only 3 jobs. That's why I removed release-0.6 jobs in this pull request.